portfolio/src/main/webapp/script.js

```diff
@@ -40,18 +44,52 @@ function showSlides() {
   for(slide of slides)
     slide.style.display = 'none';
 
-  slideIndex = slideIndex%slides.length; 
+  slideIndex = slideIndex % slides.length; 
   
   for(dot of dots)
     dot.className = dot.className.replace(" active","");
-
+  
   slides[slideIndex].style.display = "block";  
   dots[slideIndex].className += " active";
 }
 
 function showSlidesAuto()
 {
-    slideIndex+=1;
-    console.log(slideIndex);
+    slideIndex++;
     showSlides();
 }
+//change width css attribute
+function resizePhoto(photo,size)
+{
+    photo.style.width = size;
+}
+function changeSizeState(photo)
+{
+    sizeStateBig = !sizeStateBig;
+    if(sizeStateBig)
+        resizePhoto(photo,"75vw");
+    else
+        resizePhoto(photo,"60vw");
+}
+async function getData(path) {
+  // The fetch() function returns response object, and is asynchronous.
+  const response = await fetch(path);
+  //the text() function gets the text from the response asynchronously
+  const data = await response.text();
+  // When the request is complete return text from response
+  return data;
+}
+async function includeHTML(fileName)
+{
```

Comment:
Some of your functions have the opening `{` on a separate line and some don't. I've never seen the two-line style in JavaScript so I'd recommend sticking with the one-line style. Either way, any consistent style is a good style.